.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Please remove this file. :-)

Reply:
ops, sorry, not sure how that ended up there :)